tests/src/tools/ReadyToRun.TestHarness/Program.cs

```diff
@@ -203,7 +222,12 @@ private static int AnalyzeResults(ReadyToRunJittedMethods jittedMethods, string
             return StatusTestPassed;
         }
 
-        private static void RunTest(TraceEventSession session, string testArguments, out int exitCode)
+        private static string GetFullModuleName(string moduleName, string methodName)
+        {
+            return $"{moduleName}!{methodName}";
```

Comment:
Nit - while the exclamation mark is a natural choice for a WinDbg user, we might want to spend a minute thinking whether this cannot cause trouble for generic method names which may contain exclamation marks in their signatures to indicate the presence of generic arguments.